src/links/dto/create-link.dto.ts

```diff
@@ -8,5 +9,8 @@ export class CreateLinkDto {
     allow_query_components: true,
```

Comment:
**suggestion**: Consider making the error message more descriptive to help users understand why a URL is considered unsafe.